File: june15basics/inverse.cpp
#include<iostream>
#include<cmath>
using namespace std;
int main(int argc,char** argv){
    int n=215634;
    int rs=0;
    int ct=0;

    while(n!=0){
        int rem=n%10;
        ct++;
        rs+=ct*pow(10,(rem-1));
        n=n/10;
    }
    cout<<rs<<endl;
}

File: june15basics/inverse.java
import java.util.*;

public class inverse{
    public static void main(String [] args){
        int n=316245;
        int sum=0;
        int ct=0;
        while(n!=0){
            int rem=n%10;
            n=n/10;
            ct++;
            sum+=ct*Math.pow(10, rem-1);


        }
        System.out.println(sum);
    }
}